aspnetcore-webapi/IgnorePropertyInSwagger/IgnorePropertyInSwagger/Extensions/StringExtensions.cs

```diff
@@ -0,0 +1,14 @@
+namespace IgnorePropertyInSwagger.Extensions;
+
+public static class StringExtensions
+{
+    public static string ToCamelCase(this string input)
+    {
+        if (string.IsNullOrEmpty(input) || char.IsLower(input[0]))
+            return input;
+
+        var chars = input.ToCharArray();
+        chars[0] = char.ToLower(input[0]);
+        return new(chars);
```

Comment:
Always separate a `return` statement with a blank line between the `return` and the rest of the code

Reply:
done